pyproject.toml

```
@@ -12,6 +12,7 @@ dependencies = [
     "discord.py==2.4.0",
     "feedparser==6.0.11",
     "beautifulsoup4==4.12.3",
+    "sentry-sdk==2.18.0",
```

```suggestion
    "sentry-sdk==2.28.0",
```
Updating due to age of PR